packages/cozy-stack-client/src/DocumentCollection.spec.js

```diff
@@ -144,6 +152,25 @@ describe('DocumentCollection', () => {
       expect(respWithoutNext.next).toBe(false)
     })
 
+    it('should paginate results with bookmark', async () => {
+      client.fetchJSON.mockReturnValue({
+        rows: [{}, {}],
+        total_rows: 3
+      })
+      const respWithNext = await collection.all({ limit: 2, bookmark: 'book' })
+      expect(respWithNext.next).toBe(true)
+
+      client.fetchJSON.mockReturnValue({
+        rows: [{}],
+        total_rows: 3
+      })
+      const respWithoutNext = await collection.all({
+        limit: 2,
+        bookmark: 'mark'
+      })
+      expect(respWithoutNext.next).toBe(false)
+    })
+
```

Comment:
Nit: I think it's better to add the tests inside the commit adding the functionality, I was actually looking for them when I saw the implementation commit.

Reply:
I actually prefer to have a separated commit for the test, to keep the commits focus on one thing (feature, test, doc, etc), but no problem to squash that